milestone-3/frontend/src/components/RecipePage/IngredientForm/IngredientForm.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default function IngredientForm({newElement, onChange, onClick}) {
+    return (
+        <div>
+            <p><b>Add an ingredient</b></p>
+            <input 
+                placeholder="e.g. 1 cup berries"
+                value={newElement} 
+                onChange={onChange}
+            />
+            <button onClick={onClick}>Add</button>
```

Comment:
It looks like you figured out how to pass functions and state into components. Nice!